packages/wordcount/src/index.js

```diff
@@ -93,12 +93,13 @@ function matchCharacters( text, regex, settings ) {
 
 export function count( text, type, userSettings ) {
 	const settings = loadSettings( type, userSettings );
+	let returnData = 0;
```

Comment:
Instead of setting this to `0` and changing it, it might be clearer (and one less line) to just return inside the lower if... eg:

```js
export function count( text, type, userSettings ) {
	const settings = loadSettings( type, userSettings );
	if ( text ) {
		let matchRegExp = settings[ type + 'RegExp' ];
		const results = ( 'words' === settings.type ) ? matchWords( text, matchRegExp, settings ) : matchCharacters( text, matchRegExp, settings );
		if ( results ) {
			return results.length;
		}
		return 0;
	}
}